src/__tests__/SearchResults.test.js

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("search results", () => {
+    const { asFragment } = render(<SearchResults />)
+
+    it("search results renders correctly", () => {
+        expect(asFragment()).toMatchSnapshot();
+    });
+
+    it("should display images in component", () => {
+     expect(img).toHaveAttribute('spaceImage')
+    })
+
+});
```

Comment:
This isnt working for the same reasons as above, also that you dont have the needed valid props to pass into the components. Also, the second test wont work as you need to create an instance of the component to test, and to use a query to find the elements in that rendering to run your assertions on:
```
describe("search results", () => {
  const validProps = ["test-url-1", "test-url-2", "test-url-3", "test-url-4"];

  it("search results renders correctly", () => {
    const { asFragment } = render(<SearchResults results={validProps} />);

    expect(asFragment()).toMatchSnapshot();
  });

  it("should display images in component", () => {
    render(<SearchResults results={validProps} />);
    const images = screen.getAllByRole("img");

    expect(images).toHaveLength(4);
  });
});
```